src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Order.xml

```diff
@@ -102,6 +102,13 @@
             <group>admin:order:read</group>
         </attribute>
 
+        <attribute name="itemsTotal">
+            <group>admin:order:read</group>
```

Comment:
I believe that if we add only scenarios for Visitor, we should not add the additional serialization for Admin. Either we remove it or we add a new scenario from Admin's perspective.